packages/react-router/__tests__/useParams-test.tsx

```diff
@@ -11,7 +11,7 @@ import {
 describe("useParams", () => {
   describe("when the route isn't matched", () => {
     it("returns an empty object", () => {
-      let params: Record<string, string> = {};
+      let params: Record<string, string | undefined> = {};
```

Comment:
Can we reuse the `Params` type in this file instead of declaring our own `Record` type here? Not a big deal, just curious.

Reply:
Yeah totally, will make a TODO for this.